### ISSUE TITLE
Unable to save screenshot of multisample FBO

### ISSUE THREAD

**Issue by Clockwork-Muse** [panda3d/panda3d#1121]
## Description
When rendering to an FBO with multisamples enabled on EGL, it outputs a black/transparent image.

## Steps to Reproduce
```python
from direct.showbase.ShowBase import ShowBase
from panda3d.core import AmbientLight, FrameBufferProperties, GraphicsOutput, GraphicsPipe, LColor, LVector2i, Texture, WindowProperties, loadPrcFileData

class MyApp(ShowBase):
    def __init__(self):
        super().__init__(self)

        self.cam.set_pos(7, 7, 7)
        self.cam.look_at(0, 0, 0)

        window_props = WindowProperties(size=LVector2i(1024, 768))
        flags = GraphicsPipe.BFRefuseWindow

        color_props = FrameBufferProperties()
        color_props.set_rgba_bits(8, 8, 8, 0)
        color_props.set_float_color(False)
        color_props.set_depth_bits(16)
        color_props.set_float_depth(True)
        # This line causes issues
        color_props.set_multisamples(4)

        self.color_buffer = self.graphicsEngine.make_output(self.pipe, "color buffer", -1, color_props, window_props, flags, self.win.get_gsg(), self.win)
        self.color_buffer.add_render_texture(tex=Texture(), mode=GraphicsOutput.RTMBindOrCopy, bitplane=GraphicsOutput.RTPColor)
        self.color_buffer.get_texture().set_clear_color(LColor(0, 0, 0, 0))
        color_camera = self.make_camera(self.color_buffer, camName="color")
        color_camera.reparent_to(self.render)
        color_camera.set_pos(7, 7, 7)
        color_camera.look_at(0, 0, 0)

        model = self.loader.loadModel("untitled.blend")
        model.reparent_to(self.render)

        light_node = AmbientLight('light')
        light_node.set_color((0.8, 0.8, 0.5, 1))
        light = self.render.attach_new_node(light_node)
        self.render.set_light(light)


def generate_synth():
    loadPrcFileData("",
    """
        win-size 1024 768
        window-type offscreen
        model-cache-dir
    """)

    base = MyApp()
    base.graphicsEngine.render_frame()
    base.graphicsEngine.render_frame()
    base.screenshot(source=base.color_buffer, namePrefix="screenshot.png", defaultFilename=False)

generate_synth()
```

(`untitled.blend` is just a default cube)

In contrast, the scene renders just fine if multisampling is disabled:

`color_props.set_multisamples(0)`
![screenshot](https://user-images.githubusercontent.com/5569508/109334089-1bbdc500-7815-11eb-8ae1-553a787d64fa.png)

## Workaround
Note that if this buffer is used as the window for initializing simplepbr, multisampling works:
```python
simplepbr.init(
    use_normal_maps=True,
    enable_shadows=True,
    use_occlusion_maps=True,
    window=self.__color_buffer,
    camera_node=color_camera,
    msaa_samples=4
)
```

... so it's possible I'm just misunderstanding something.

## Environment
 - Operating system: Ubuntu 20.04
 - Docker Container base: nvidia/opengl:1.2-glvnd-runtime-ubuntu20.04
 - System architecture: x86_64
 - Panda3D version: 1.11.0
 - Installation method: built from source, commit ada74e3b0c35a12d5efb3b5bcbdb38642c4e7679
 - Python version: 3.8.5

## Related
#1089

**Comment by Clockwork-Muse** [comment]
@rdb

**Comment by rdb** [comment]
This does not appear to be related to EGL, because I get the same issue with GLX.

I think the issue is whatever makes the screenshot—it's binding the multisample FBO rather than the resolved FBO.

**Comment by rdb** [comment]
Fixed in 5cadd86d1edf86182347d4d14b6f95cf108aa57c
